AccentColorizer/AccentColorHelper.h

```diff
@@ -2,6 +2,16 @@
 #include "framework.h"
 
 extern COLORREF g_dwAccent;
-extern int g_hsvAccentH;
+extern int g_hslAccentH;
+extern double g_hslAccentS;
+extern double g_oldhslAccentS;
+extern double g_balance1hslAccentS;
+extern double g_balance2hslAccentS;
+extern double g_hslAccentL;
+extern double g_oldhslAccentL;
+extern double g_defaulthslAccentH;
```

Comment:
It may be cleaner to store those 3 variables (`g_defaulthslAccentH`, `g_defaulthslAccentS` and `g_defaulthslAccentL`) as `struct hsl_t`